src/main/java/com/sourcegraph/cody/MyAccountTabUtils.kt

```diff
@@ -28,25 +26,12 @@ fun fetchMyAccountPanelData(
 
     if (activeAccountType.isDotcomAccount()) {
       ApplicationManager.getApplication().executeOnPooledThread {
-        val codyProFeatureFlag =
-            server
-                .evaluateFeatureFlag(GetFeatureFlag.CodyProJetBrains)
-                .completeOnTimeout(false, 4, TimeUnit.SECONDS)
-                .get() == true
-        if (codyProFeatureFlag) {
-          val isCurrentUserPro = getIsCurrentUserPro(server) ?: false
-          result.complete(
-              MyAccountTabPanelData(
-                  activeAccountType.isDotcomAccount(),
-                  codyProFeatureFlag = true,
-                  isCurrentUserPro = isCurrentUserPro))
-        } else {
-          result.complete(
-              MyAccountTabPanelData(
-                  activeAccountType.isDotcomAccount(),
-                  codyProFeatureFlag = false,
-                  isCurrentUserPro = null))
-        }
+        val isCurrentUserPro = getIsCurrentUserPro(server) ?: false
+        result.complete(
+            MyAccountTabPanelData(
+                activeAccountType.isDotcomAccount(),
+                codyProFeatureFlag = true,
```

Comment:
I guess we can remove it here too - it should be always true now.

Reply:
Good point, fixed.